bindgen-tests/tests/expectations/tests/reference_newtype_wrapper.rs

```diff
@@ -0,0 +1,23 @@
+#![allow(dead_code, non_snake_case, non_camel_case_types, non_upper_case_globals)]
+#[repr(transparent)]
+pub struct __bindgen_marker_Reference<T: ?Sized>(*const T);
```

Comment:
Can you add some tests for these in structs? In particular reference members in structs are not that uncommon.

I _think_ this as-is breaks `#[derive]` and such, but maybe it's not a huge issue since it's opt-in. At least we should file an issue and reference it from the code that generates these wrappers.

Reply:
Ah, I guess this is only a test issue in practice, since the way this is implemented leaves the "reference" type to the consumer.

So seems fine with this changed to `(T)`, _or_ with the reference wrapping the inner thing (which might be more correct / flexible? You might want to e.g. make something like `struct Reference<T>(NonNull<T>)`, and you otherwise wouldn't be able to.